core-js/src/main/javascript/cdf/components/ccc/BaseCccComponent.js

```diff
@@ -100,7 +100,7 @@ define([
       }
 
       // clear placeholder
-      var ph = $("#" + this.htmlObject).empty();
+      var ph = !!this.cleanElementBeforeRender ? $("#" + this.htmlObject).empty() : $("#" + this.htmlObject);
```

Comment:
No need for the `!!` because we're directly on an "if" context.

Reply:
Thanks for the tip